src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson. herokuapp.com/create-application"
+      method="post"
+      class="search_bar"
+      data-qa="big"
+    >
+      <label class="search_bar_label">
+        <input
+          type="text"
+          name="barBig"
+          class="search_bar_input search_barBig_input"
+          placeholder="Try &#34;Los Angeles&#34;"
+          data-qa="keypress"
+        >
+        <img
+          class="image_findBig"
+          src="images/Search.svg"
+          alt="image_find"
+        >
+      </label>
+    </form>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="search_bar"
+      data-qa="small"
+    >
```

Comment:
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
      class="search_bar"
      data-qa="small"
    >
```

Reply:
thank you for the correction
I will keep it in mind